python/configuration_manager/configurations_manager.py

```diff
@@ -105,9 +84,9 @@ def load_log_configurations(self, name, directory=None,
         return logger.initialize_logger(name, target_directory,
                                         configurations=log_configurations)
 
-
 if __name__ == '__main__':
     # configure logger
-    myLogger = ConfigurationsManager().load_log_configurations(__name__)
+    my_logger = ConfigurationsManager().load_log_configurations(__name__)
```

Comment:
It could be better to establish a way to call objects and/or local variables, e.g. l_logger, logger_obj, cm_logger. or simple create the Object from ConfigurationManager and than qualify the .load_log_configuration method.